modules/xfem/doc/content/source/reporters/StressCorrosionCracking.md

```diff
@@ -0,0 +1,49 @@
+# StressCorrosionCracking
+
+## Description
+
+This Reporter computes the growth increment for each active crack front node in the [CrackMeshCut3DUserObject.md].  The growth increment for each node is computed by first finding the crack front node where $K_I$ provided by the [DomainIntegralAction.md] is largest and computing the amount of time, $t_{cmax}$, it will take for that node to extend by the amount given by [!param](/Reporters/StressCorrosionCracking/max_growth_size).  $t_{cmax}$ is stored in the scalar reporter [!param](/Reporters/StressCorrosionCracking/time_to_max_growth_size_name).  The growth increment for the other nodes in the crack front are then computed by multiplying their growth rates by $t_{cmax}$.  The growth rate comptuted by `StressCorrosionCracking` is based on the equation for stress corrosion cracking given by [!cite](li_scc_2015) for stainless steel 316L exposed to water at 288C given by
```

Comment:
Is the node with the highest $K_I$ always the one with the largest growth increment?

Reply:
Yes and its growth increment will be equal to max_growth_size